src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -15,6 +15,7 @@ namespace Microsoft.PowerShell.Commands
     /// The ConvertTo-Json command.
```

Comment:
@iSazonov, your last commit had 4 failures in `PowerShell-CI-linux`
Json Bug fixes.ConvertTo-Json -Depth 100 throws JsonException (CycleDetected) when the user specifies a depth greater or equal than 100.
```powershell
Expected strings to be the same, but they were different.
Expected length: 54
Actual length:   42
Strings differ at index 5.
Expected: 'ThrowInval...'
But was:  'ThrowJsonE...'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Json.Tests.ps1: line 1419
1419:                     $exc.Exception.TargetSite.Name | Should -BeExactly $testCase.TargetSiteName
```
ConvertTo-Json.Should handle null
```powershell
Value cannot be null. (Parameter 'inputType')
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Json.Tests.ps1: line 88
88:         $null | ConvertTo-Json -Compress | Should -Be 'null'

```
ConvertTo-Json.Parameter works: -Depth <depth>
```powershell
Expected strings to be the same, but they were different.
Expected length: 54
Actual length:   42
Strings differ at index 5.
Expected: 'ThrowInval...'
But was:  'ThrowJsonE...'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Json.Tests.ps1: line 135
135:         $exc.Exception.TargetSite.Name | Should -BeExactly "ThrowInvalidOperationException_SerializerCycleDetected"
```
ConvertTo-Json.Cycle detection works
```powershell
Expected strings to be the same, but they were different.
Expected length: 54
Actual length:   42
Strings differ at index 5.
Expected: 'ThrowInval...'
But was:  'ThrowJsonE...'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Json.Tests.ps1: line 205
205:         $exc.Exception.TargetSite.Name | Should -BeExactly "ThrowInvalidOperationException_SerializerCycleDetected"
```